homeassistant/const.py

```diff
@@ -164,6 +164,7 @@
 CONF_SSL = "ssl"
 CONF_STATE = "state"
 CONF_STATE_TEMPLATE = "state_template"
+CONF_STEP_DESCRIPTION = "description"
```

Comment:
Originally considered "comment", as the use-case felt a lot like code comments. 

But we're already using "description" fields for describing scripts and automations, so figured I should stay consistent.